code/modules/mob/living/simple_animal/bot/ssdbot.dm

```diff
@@ -0,0 +1,262 @@
+
+#define SIGH 0
+#define ANNOYED 1
+#define DELIGHT 2
+
+/mob/living/simple_animal/bot/mulebot/ssdbot
+	name = "\improper SSD retrieval bot"
+	desc = "A SSD-Retrieval bot."
+	icon = 'icons/vehicles/CargoTrain.dmi'
+	icon_state = "ambulance"
+	var/turf/home_turf = null
+	var/idle_cycles = 0
+	var/move_steps = 5
+	locked = 1
+	remote_disabled = 1
+	allow_pai = 0
+	report_delivery = 0
+	bot_core_type = /obj/machinery/bot_core/mulebot/ssdbot
+	health = 250
+	maxHealth = 250
+	var/list/ignored_ssds = list()
+	radio_channel = "Supply"
+	var/trapped = 0
+	var/trappedcycles = 0
+	var/speaks = 0 // debug setting.
+	var/soundeffects = 0
+	var/automatic_mode = 1 // if 1, players cannot reconfigure it
+	buckle_lying = 1
+	force_threshold = 20
+	var/list/areas_to_ignore = list(/area/medical/virology, /area/toxins/xenobiology, /area/security/prison, /area/security/permabrig)
+	var/pickup_delay = 60 // 60 cycles, 120 seconds
+	var/list/previous_targets = list()
+	var/mob/targetssd = null
+
+/mob/living/simple_animal/bot/mulebot/ssdbot/New()
+	..()
+	name = initial(name)
+	home_turf = get_turf(src)
+	var/datum/job/captain/J = new/datum/job/captain
```

Comment:
What the actual fuck?
This should just be `access_card.access = get_all_accesses()`
You should never directly create a job datum.